src/com/yetanalytics/lrs_admin_ui/views/reactions.cljs

```diff
@@ -340,13 +343,14 @@
                     :label "Boolean OR"}
                    {:value :not
                     :label "Boolean NOT"}]
```

Comment:
Could these labels be translated? I think "Statement Criteria" (not shown because GH is stupid) should be translated, but the "Boolean [AND/OR/NOT]" labels are obvious and technical enough to be left as-is.

Reply:
yeah i left boolean because its VERY tricky to translate because of more combinatory and reused stuff in the views. ALSO because they are operations. Statement Criteria I am thinking as on the same level as the ops. Kinda like how when you program in another language you still use the same keywords, this is literally programming.